src/util/contribution-types.js

```diff
@@ -99,6 +99,10 @@ const defaultTypes = function(repoType) {
       symbol: '📹',
       description: 'Videos',
     },
+    maintenance: {
```

Comment:
this change has in pr https://github.com/all-contributors/all-contributors-cli/pull/142

Reply:
I've removed that and the line on the README file.